templates/focus-ios.xliff

```diff
@@ -139,7 +139,7 @@
       </trans-unit>
       <trans-unit id="Autocomplete.addCustomUrlPlaceholder">
         <source>Paste or enter URL</source>
-        <note>Label for button to add a custom URL</note>
+        <note>Watermark for the input field to add a custom URL</note>
```

Comment:
Watermark -> https://en.wikipedia.org/wiki/Watermark
I guess you meant placeholder here? In case, just an FYI for next export.